app/utils/scorecard_util.js

```diff
@@ -34,4 +35,13 @@ export const getLocationMaxWidth = (scorecard) => {
     return locationLength * (scorecard.primary_school != null ? getDeviceStyle(wp('0.8%'), wp('0.6%')) : getDeviceStyle(wp('1%'), wp('0.8%')));
 
   return getDeviceStyle(locationLength * wp('3.8%'), locationLength * wp('1%'))
+}
+
+export const handleScorecardCodeClipboard = async () => {
+  let copiedText = await Clipboard.getString();
+
+  if (!parseInt(copiedText)) {
+    copiedText = copiedText.slice(-6);
```

Comment:
@limkimsan this code didn't detect to apply only for the valid URL of the scorecard URL but all with just extract the last 6 digits, we should prevent to paste only 6 digits or the scorecard code full URL only

Reply:
@kakada , noted bong. I will check for the valid URL.